zio-http-benchmarks/src/main/scala/Main.scala

```diff
@@ -0,0 +1,26 @@
+import io.netty.handler.codec.http.HttpHeaderNames
+import io.netty.util.AsciiString
+import zhttp.http.{Response, _}
+import zhttp.service.server.ServerChannelFactory
+import zhttp.service.{EventLoopGroup, Server}
+import zio.{App, ExitCode, URIO}
+
+case class Message(message: String)
+
+object Main extends App {
```

Comment:
- Move this to `Example` project
- Rename to `PlainTextBenchmarkServer`
- Add a scala-doc to explain what this server does.